modules/llm-cache/storage/file_storage.cc

```diff
@@ -189,15 +199,24 @@ Status FileStorage::Update(
     }
   }
 
+  std::lock_guard<std::mutex> lock(gcMutex);
   int j = 0;
   for (size_t i = 0; i < pathList.size(); i++) {
     if (pathIndexMap.find(i) != pathIndexMap.end()) {
       j += 1;
+      if (createFileSet.find(this->rootPath + pathList[i]) !=
+          createFileSet.end()) {
+        TouchFile(this->rootPath + pathList[i]);
+        gcList.push_back(this->rootPath + pathList[i]);
+      }
+    } else {
+      lock.~lock_guard();
+      break;
     }
   }
```

Comment:
Use lexical scope rather than explicit dtor, i.e.,

```suggestion
  {
    std::lock_guard<std::mutex> lock(gcMutex);
    int j = 0;
    for (size_t i = 0; i < pathList.size(); i++) {
      if (pathIndexMap.find(i) != pathIndexMap.end()) {
        j += 1;
        if (createFileSet.find(this->rootPath + pathList[i]) !=
            createFileSet.end()) {
          TouchFile(this->rootPath + pathList[i]);
          gcList.push_back(this->rootPath + pathList[i]);
        }
      } else {
        break;
      }
    }
  }
```

Reply:
OK, I understand.